docs/usage.rst

```diff
@@ -253,6 +253,36 @@ For more information, see the
 section in the documentation of the 'zhmcclient' package.
 
 
+Communication with Prometheus
+-----------------------------
+
+The exporter is an HTTP or HTTPS server that is regularly contacted by Prometheus
+for collecting metrics using HTTP GET.
+
+The parameters for the communication with Prometheus are defined in the
+HMC credentials file in the ``prometheus`` section, as in the following example:
+
+.. code-block:: yaml
+
+    prometheus:  # optional
+      port: 9291
+      scheme: https
```

Comment:
@andy-maier Just want to simplify the mTLS related parameters. Could you think about omitting the "scheme"? Instead, by default use HTTP and enable HTTPS if "server_cert_file" and "server_key_file" settings are provided.

Reply:
DONE. The update removes "scheme" and implies HTTPS from the presence of "server_cert_file".